docs/guide/writing/beginners-guide-to-docs.rst

```diff
@@ -1,323 +1,210 @@
-documentation serves the same purpose.
-
-.. sidebar::  Sidebar on open source
-
-	There is a magical feeling that happens when you release your code.
-	It comes in a variety of ways, but it always hits you the same.
-	*Someone is using my code?!*
-	A mix of terror and excitement.
-
-		| I made something of value!
-		| What if it breaks?!
-		| I am a real open source developer!
-		| Oh god, someone else is using my code..
-
-	Writing good documentation will help alleviate some of these fears.
-	A lot of this fear comes from putting something into the world.
-	My favorite quote about this is something along these lines:
-
-		| Fear is what happens when you're doing something important.
-		| If you are doing work that isn't scary,
-		| it isn't improving you or the world.
-
-	Congrats on being afraid!
-	It means you're doing something important.
-
-You want people to use your code
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-You have written a piece of code,
-and released it into the world.
-You have done this because you think that others might find it useful.
-However,
-people need to understand why your code might be useful for them,
-before they decide to use it.
-Documentation tells people that this project is for them.
-
-	| If people don't know why your project exists,
-	| they won't use it.
-	| If people can't figure out how to install your code,
-	| they won't use it.
-	| If people can't figure out how to use your code,
-	| they won't use it.
-
-There are a small number of people who will source dive and use any code out there.
-That is a vanishingly small number of people,
-compared to people who will use your code when properly documented.
-If you really love your project,
-document it,
-and let other people use it.
-
-
-You want people to help out
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Open source is this magical thing right?
-You release code,
-and the code gnomes come and make it better for you.
-
-Not quite.
-
-There are lots of ways that open source is amazing,
-but it doesn't exist outside the laws of physics.
-You have to put work in,
-to get work out.
-
-	| You only get contributions after you have put in a lot of work.
-	| You only get contributions after you have users.
-	| You only get contributions after you have documentation.
-
-Documentation also provides a platform for your first contributions.
-A lot of people have never contributed before,
-and documentation changes are a lot less scary than code changes.
-If you don't have documentation,
-you will miss out on a whole class of contributors.
-
-You want your code to be better
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* They don't know why your project exists or how it meets their needs.
+* They can't find how to install your code.
+* They can't see how to use your code.
 
-It's really easy to have an idea in your head that sounds perfect,
-but the act of putting words to paper requires a distillation of thought that may not be so easy.
+Few people will source dive to use code, but most 
```

Comment:
rephrase "source dive" maybe?

Reply:
And I feel that document your project is different to document your code, but :man_shrugging: